### DISCUSSION TITLE
Plotting segmentation masks on input images

### DISCUSSION THREAD

**Discussion by mfaizan-ai** [Project-MONAI/MONAI#8226] [General]
Are there any monai function that overlay segmentation masks which (1, 3, 128, 128, 128) corresponding to (Batch, channels, depth, height, width) on the input image with shape (1, 4, 128, 128, 128).  I am doing segmentation on BraTS 2023 dataset. And I have to put qualitative results in my paper. Please let me know if there is any tutorial on monai that does it.

**Comment by KumoLiu** [comment]
Hi @faizan1234567, thanks for your interest here.
You can refer to this tutorial: https://github.com/Project-MONAI/tutorials/blob/main/modules/transform_visualization.ipynb

Hope it helps, thanks.
(Move to discussion since not a issue)

**Comment by mfaizan-ai** [comment]
@KumoLiu  thank you for the assistance. I believe this tutorial is very simple and does not cover how to blend segmentation masks on the whole MRI image or a slice for analysis and visualization.
